### PR TITLE
1.20.1 patches

### PR DESCRIPTION
**Pull request by Dweblenod**
Misc fixes

### CODE REVIEW

File: src/main/java/com/mraof/minestuck/data/tag/MinestuckBlockTagsProvider.java [hunk 1]
```diff
@@ -150,20 +150,11 @@ protected void addTags(HolderLookup.Provider provider)
...
-		tag(NEEDS_STONE_TOOL).add(UNCARVED_WOOD.get(), UNCARVED_WOOD_BUTTON.get(), UNCARVED_WOOD_FENCE.get(), UNCARVED_WOOD_FENCE_GATE.get(), UNCARVED_WOOD_PRESSURE_PLATE.get(), UNCARVED_WOOD_SLAB.get(),
+		tag(MINEABLE_WITH_AXE).add(CARVED_PLANKS.get(), CARVED_BUTTON.get(), CARVED_FENCE.get(), CARVED_FENCE_GATE.get(), CARVED_PRESSURE_PLATE.get(), CARVED_SLAB.get(), CARVED_STAIRS.get(), WOODEN_GRASS.get(),
+				TREATED_PLANKS.get(), TREATED_BUTTON.get(), TREATED_FENCE.get(), TREATED_FENCE_GATE.get(), TREATED_PRESSURE_PLATE.get(), TREATED_PLANKS_SLAB.get(), TREATED_PLANKS_STAIRS.get(),
+				TREATED_WOODEN_GRASS.get(), LACQUERED_PLANKS.get(), LACQUERED_BUTTON.get(), LACQUERED_FENCE.get(), LACQUERED_FENCE_GATE.get(), LACQUERED_PRESSURE_PLATE.get(),
+				LACQUERED_SLAB.get(), LACQUERED_STAIRS.get(), LACQUERED_WOODEN_MUSHROOM.get(), WOODEN_LAMP.get());
+		needsStoneAxe(UNCARVED_WOOD.get(), UNCARVED_WOOD_BUTTON.get(), UNCARVED_WOOD_FENCE.get(), UNCARVED_WOOD_FENCE_GATE.get(), UNCARVED_WOOD_PRESSURE_PLATE.get(), UNCARVED_WOOD_SLAB.get(),
```
Comment (Dweblenod) [inline src/main/java/com/mraof/minestuck/data/tag/MinestuckBlockTagsProvider.java:157]: The way it is now has no net effect in data files. I think it helps illustrate some peculiarities/inconsistencies with how the tags are handled for these blocks

File: CHANGELOG.md [hunk 2]
```diff
@@ -12,11 +12,17 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/).
 - Frost Trees will now drop saplings
 - Saplings are now pottable
 - Aspect slabs can now be used in crafting recipes
+- Uranium button now made with 1 raw uranium instead of 1 uranium block
+- Removed default name visibility of Lotus Flower
```
Comment (kirderf1) [inline CHANGELOG.md:16]: I recall seeing this issue in 1.20.4, but was this also an issue back in 1.20.1?

File: CHANGELOG.md [hunk 3]
```diff
@@ -12,11 +12,17 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/).
...
 
 ### Fixed
 
 - Fix player gutter multiplier, which wasn't being saved
 - Space sapling texture baseline lowered, space and frost sapling sides trimmed
+- Added null check for target in MobAnimationPhaseGoal
+- Changed bookshelf recipes to match their book drop rate
```
Comment (kirderf1) [inline CHANGELOG.md:23]: The recipes still need changing for this.

**Review by kirderf1 — CHANGES_REQUESTED** [review]
Since this is targeting 1.20.1, how much of this is backports from the main branch, and how much also need to be fixed on the main branch?

**Comment by Dweblenod** [comment]
Regarding the two comments above on Minecraft version, everything present in this branch should be an issue in 1.20.4 although in messaging prior to this PR some of them have been stated as being 1.20.4 specific. I may have gotten versions mixed up or noticed the situation first when testing 1.20.4

File: CHANGELOG.md [hunk 4]
```diff
@@ -12,11 +12,17 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/).
...
 - Saplings are now pottable
 - Aspect slabs can now be used in crafting recipes
+- Uranium button now made with 1 raw uranium instead of 1 uranium block
+- Removed default name visibility of Lotus Flower
 
 ### Fixed
 
 - Fix player gutter multiplier, which wasn't being saved
 - Space sapling texture baseline lowered, space and frost sapling sides trimmed
+- Added null check for target in MobAnimationPhaseGoal
```
Comment (kirderf1) [inline CHANGELOG.md:22]: Should be written to something more meaningful to the players.

File: CHANGELOG.md (hunk 2 above)
Comment (kirderf1) [reply CHANGELOG.md:16]: Tested it out just now, and the 1.20.4 issue does not appear to be present in 1.20.1 with Minestuck only. Unless there's a specific reason to this for 1.20.1, I think that we should revert this and the `LotusFlowerEntity` change before merging this.

**Review by kirderf1 — APPROVED** [review]

**Pull request merged**